build-logic/src/main/kotlin/dokkabuild.testing.android-setup.gradle.kts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2014-2023 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+import org.gradle.api.tasks.PathSensitivity.RELATIVE
+
+/**
+ * Utilities for preparing Android projects for Integration Tests
+ */
+
+plugins {
+    base
+}
+
+/**
+ * Create an `local.properties` file so that Android projects will automatically work in integration tests.
+ */
+abstract class CreateAndroidLocalPropertiesFiles : DefaultTask() {
+    @get:OutputFiles
+    val localPropertiesFilesDestinations: List<File>
```

Comment:
Sure thing, done. I've updated it to a `Provider<Set<File>>` (which `SetProperty<File>` implements) because I think using `Provider<>` instead of `Property<>` makes it more clear that this is a read-only value.